src/seedu/addressbook/data/person/Phone.java

```diff
@@ -36,10 +33,6 @@ public static boolean isValidPhone(String test) {
         return test.matches(PHONE_VALIDATION_REGEX);
```

Comment:
Do you notice that the `isValid` is quite repetitive if placed in child classes? 

Hint: It requires to write your own constructor in Parent rather than having the default no-argument one.